demisto_sdk/commands/format/update_job.py

```diff
@@ -41,15 +42,9 @@ def attempt_infer_selected_feeds(self):
     def run_format(self) -> int:
         try:
             click.secho(f'\n======= Updating file: {self.source_file} =======', fg='white')
+            super().update_json(default_from_version=FILETYPE_TO_DEFAULT_FROMVERSION.get(FileType.JOB))
             self.update_id()
             self.attempt_infer_selected_feeds()
-            self.set_from_server_version_to_default()
-
-            # relevant parts of super().update_json()
-            self.remove_null_fields()
-            self.remove_unnecessary_keys()
-            self.remove_spaces_end_of_id_and_name()
-
```

Comment:
is this intentional?

Reply:
Already calling this methods on update_json.
Duplication.